app/config/featureSwitch/Features.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config.featureSwitch
+
+import javax.inject.{Inject, Singleton}
+import play.api.Configuration
+
+@Singleton
+class Features @Inject()(implicit config: Configuration){
```

Comment:
Can you add in a quick test spec for the feature switching functionality. Check view vat returns FE for a simple example